rollingpin/simpleflake.py

```diff
@@ -0,0 +1,89 @@
+# ========================================================================
+# simple-flake - https://github.com/SawdustSoftware/simple-flake
+# ========================================================================
+
+# Copyright (c) 2013 CustomMade Ventures
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import time
+import random
+import collections
+
+#: Epoch for simpleflake timestamps, starts at the year 2000
+SIMPLEFLAKE_EPOCH = 946702800
+
+#field lengths in bits
+SIMPLEFLAKE_TIMESTAMP_LENGTH = 41
+SIMPLEFLAKE_RANDOM_LENGTH = 23
+
+#left shift amounts
```

Comment:
these comments are breaking linting because a space is missing after `#`

Reply:
the broken tests are from earlier stuff.